README.md

```diff
@@ -103,6 +106,10 @@ We use a common [Code of Conduct](https://github.com/filecoin-project/community/
 
 Each repo should have its own contributing guide, called `CONTRIBUTING.md`. Here is an example [Contributing Guide for `go-filecoin`](https://github.com/filecoin-project/go-filecoin/blob/master/CONTRIBUTING.md).
 
+### Filecoin Shipyard
```

Comment:
Thanks for creating this section @michellebrous! This shouldn't be in events though. Can we create a new section called "## Contribute" (in the place where "## Contributing Guidelines" currently is) and include both "### Filecoin Shipyard" and the existent "### Contributing Guidelines" to it?

If you add a section in the README (which this would entail), can you also please edit the Table of Contents at the top to reflect the new sections?

Thanks!

Reply:
Ahh not sure how it ended up under events. Fixed!